sites/offshore-mag.com/server/templates/subscribe/index.marko

```diff
@@ -1,74 +1,40 @@
-import GAM from "../../../config/gam";
+$ const { config } = out.global;
 
-$ const { config, site } = out.global;
-$ const dragonForms = require('../../../config/dragon-forms');
```

Comment:
@Alixers505 This needs to stay.  as it is used in the file.

Reply:
ack, yup. The rollup I was looking to for reminders on what to change was done before I used the dragonforms for the subscribe links.